sqdtoolz/Experiments/Experimental/ExpRabi.py

```diff
@@ -1,18 +1,22 @@
+from math import pi
 from sqdtoolz.Experiment import*
 from sqdtoolz.HAL.WaveformGeneric import*
 from sqdtoolz.HAL.WaveformSegments import*
 from sqdtoolz.Utilities.DataFitting import*
 from sqdtoolz.Experiments.Experimental.ExpCalibGE import*
 
 class ExpRabi(Experiment):
-    def __init__(self, name, expt_config, wfmt_qubit_drive, range_amps, SPEC_qubit, transition='GE', iq_indices = [0,1], **kwargs):
```

Comment:
While you are doing MR (merge requests , i would suggest adding doc-strings so later automate documentation generation could be introduced. Easier to maintain too.